tensorflow_addons/activations/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I like breaking these out into their own subpackage. Though this directory will need a README that covers contributing guidelines (can be pretty minimal for now).